webapp/src/main/resources/public/js/stores/branches/BranchesHistoryStore.js

```diff
@@ -45,6 +46,20 @@ class BranchesHistoryStore {
         this.openBranchStatistic = openBranchStatistic;
     }
 
+    changeCreatedBefore(createdBefore) {
+        this.createdBefore = createdBefore
```

Comment:
all other methods are simple setters so this should stay a simple setter as well. What call sites needs to be updated to mimic that behavior if we're keeping a normal setter?

Reply:
Was able to simplify these setters by handling the value as strings and letting the Datetime component convert to the moment object and handle the parsing (which it already did by default 😅 )